From andrew@lunn.ch  Tue Jan  5 14:40:33 2010
From: Andrew Lunn <andrew@lunn.ch>
Date: Sat,  2 Jan 2010 11:30:38 +0100
Subject: Staging: batman-adv: Replace KERN_DEBUG with bat_dbg
To: gregkh@suse.de
Cc: b.a.t.m.a.n@lists.open-mesh.net, Andrew Lunn <andrew@lunn.ch>
Message-ID: <1262428252-26439-3-git-send-email-andrew@lunn.ch>


Ameya Palande requested we replace printk(KERN_DEBUG "") by pr_debug()
I decided it was better to use our debug macro.

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
Acked-by: Marek Lindner <lindner_marek@yahoo.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/batman-adv/TODO     |    6 ------
 drivers/staging/batman-adv/device.c |    6 +++---
 2 files changed, 3 insertions(+), 9 deletions(-)

--- a/drivers/staging/batman-adv/device.c
+++ b/drivers/staging/batman-adv/device.c
@@ -207,7 +207,7 @@ ssize_t bat_device_write(struct file *fi
 	struct batman_if *batman_if;
 
 	if (len < sizeof(struct icmp_packet)) {
-		printk(KERN_DEBUG "batman-adv:Error - can't send packet from char device: invalid packet size\n");
+		bat_dbg(DBG_BATMAN, "batman-adv:Error - can't send packet from char device: invalid packet size\n");
 		return -EINVAL;
 	}
 
@@ -218,12 +218,12 @@ ssize_t bat_device_write(struct file *fi
 		return -EFAULT;
 
 	if (icmp_packet.packet_type != BAT_ICMP) {
-		printk(KERN_DEBUG "batman-adv:Error - can't send packet from char device: got bogus packet type (expected: BAT_ICMP)\n");
+		bat_dbg(DBG_BATMAN, "batman-adv:Error - can't send packet from char device: got bogus packet type (expected: BAT_ICMP)\n");
 		return -EINVAL;
 	}
 
 	if (icmp_packet.msg_type != ECHO_REQUEST) {
-		printk(KERN_DEBUG "batman-adv:Error - can't send packet from char device: got bogus message type (expected: ECHO_REQUEST)\n");
+		bat_dbg(DBG_BATMAN, "batman-adv:Error - can't send packet from char device: got bogus message type (expected: ECHO_REQUEST)\n");
 		return -EINVAL;
 	}
 
--- a/drivers/staging/batman-adv/TODO
+++ b/drivers/staging/batman-adv/TODO
@@ -29,12 +29,6 @@
 -> PRIMARY)
 
 => logging
-* the log level LOG_TYPE_CRIT, LOG_TYPE_WARN & LOG_TYPE_NOTICE will be
-* unified to use printk
-* LOG_TYPE_BATMAN & LOG_TYPE_ROUTES will also use printk but only after the
-* internal debug level has been raised
-* the internal debug level can be modified using a module parameter (debug)
-* or at run time via /sys/module/batman-adv/parameters/debug
 * make use of printk %pM support instead of converting mac addresses
 * manually
 
